python/pyspark/pandas/tests/indexes/test_datetime.py

```diff
@@ -101,6 +104,28 @@ def test_properties(self):
                 self.assert_eq(psidx.day_of_year, pidx.day_of_year)
                 self.assert_eq(psidx.day_of_week, pidx.day_of_week)
 
+        if LooseVersion(pd.__version__) >= LooseVersion("2.0.0"):
```

Comment:
why do we need this now? I thought you're going to address the different results in a separate tickets.